unit-tests/syncer/test-ts-desync.py

```diff
@@ -0,0 +1,89 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import log, test
+import sw
+
+
+# The timestamp jumps are closely correlated to the FPS passed to the video streams:
+# syncer expects frames to arrive every 1000/FPS milliseconds!
+sw.fps_c = sw.fps_d = 30
+sw.init( syncer_matcher = rs.matchers.dic_c )
+sw.start()
+
+#############################################################################################
+#
+test.start( "Init" )
+
+# It can take a few frames for the syncer to actually produce a matched frameset (it doesn't
+# know what to match to in the beginning)
+
+# D  C  @timestamp  comment
+# -- -- ----------- ----------------
+# 0     @0          so next expected frame timestamp is at 0+16.67
+#    0  @0
+#
+sw.generate_depth_and_color( frame_number = 0, timestamp = 0 )
+sw.expect( depth_frame = 0 )                          # syncer doesn't know about C yet, so releases right away
+sw.expect( color_frame = 0, nothing_else = True )     # no hope for a match: D@0 is already out, so it's released
+#
+# The syncer now knows about both streams, and is empty -- that was what we wanted
+
+test.finish()
+#
+#############################################################################################
+#
+test.start( "Go past C.NE" )
```

Comment:
C.NE means Color.next_expected?
I say lets write it :)

Reply:
Changed :)